src/ompl/base/OptimizationObjective.h

```diff
@@ -33,6 +33,7 @@
 *********************************************************************/
 
 /* Author: Luis G. Torres, Ioan Sucan */
+/* Edited by: Jonathan Gammell (allocInformedStateSampler) */
```

Comment:
In general, we do not use the "edited by". Just add your name to the list of authors when you make non-trivial edits.